build.gradle.kts

```
@@ -62,6 +65,9 @@ dependencies {
 	implementation("com.auth0:java-jwt:$javaJwtVersion")
 	implementation("com.auth0:jwks-rsa:$jwksRsaVersion")
 
+	// Forced dependecy cause the imported one from spring-cloud-starter-openfeign has a CVE
```

let's keep generic this section, in order to put here all the forced dependecies
```suggestion
	// Forced transient dependecies to solve CVEs
```